gui/design.lua

```diff
@@ -1475,6 +1475,27 @@ function Design:onInput(keys)
             self.placing_mirror = false
             self.needs_update = true
         else
+            -- Clicking center point
+            if #self.marks > 0 then
+                local center = self.shape:get_center()
+                if pos == center and not self.prev_center then
+                    self.start_center = pos
+                    self.prev_center = pos
+                    return true
+                elseif self.prev_center then
+                    --If there was no movement presume user wanted to click the mark underneath instead and let the flow through.
+                    if pos == self.start_center then
```

Comment:
same here. you probably want `same_xyz`

Reply:
I am reasonably sure that `self.start_center` will also be a `Point` here and make use of Coscker's handy compare function,  but once again, I am not that well-versed in this and may have misunderstood how this usage works.